test/built-ins/Promise/race/not-a-constructor.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2021 Sergey Rubanov. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-ecmascript-standard-built-in-objects
+description: >
+  Promise.race does not implement [[Construct]], is not new-able
+info: |
+  ECMAScript Function Objects
+
+  Built-in function objects that are not identified as constructors do not
+  implement the [[Construct]] internal method unless otherwise specified in
+  the description of a particular function.
+
+  sec-evaluatenew
+
+  ...
+  7. If IsConstructor(constructor) is false, throw a TypeError exception.
+  ...
+includes: [isConstructor.js]
+features: [Reflect.construct, arrow-function]
+---*/
+
+assert.sameValue(isConstructor(Promise.race), false, 'isConstructor(Promise.race) must return false');
+
+assert.throws(TypeError, () => {
+  new Promise.race([]);
+}, '`new Promise.race([])` throws TypeError');
+
```

Comment:
this test file seems redundant as you already have `test/built-ins/Promise/race/anySettled.js`. It doesn't hurt one way or another.